lib/math/polygon2d.js

```diff
@@ -135,7 +135,7 @@ export class Polygon {
     // Step 2: Check the u values of our intersections, if any are ≈0, then we
     // hit a vertex, which means that we'll double-count our crossings.
     const numberOfCrossings = intersectionResults
-        .reduce((agg, r) => agg + (r && r.u > 0.0001 ? 1 : 0), 0);
+        .reduce((agg, r) => agg + (r && r.u > 0.00001 ? 1 : 0), 0);
```

Comment:
comment this since you had to change it.

Reply:
Done.